open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/utils/VertexLabelsMainGraph.java

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.openconnectors.governancedaemonconnectors.openlineageconnectors.janusconnector.utils;
+
+public enum VertexLabelsMainGraph {
+    NODE_LABEL_COLUMN,
+    NODE_LABEL_TABLE,
+    NODE_LABEL_GLOSSARYTERM,
+    NODE_LABEL_PROCESS,
+    NODE_LABEL_SUB_PROCESS
```

Comment:
Already exists in GraphConstants but unless I am mistaken they are not being removed there?